docs/src/language_servers.md

```diff
@@ -221,6 +213,28 @@ Follow installation instructions on [LSP-html](https://github.com/sublimelsp/LSP
 
 Follow installation instructions on [LSP-jdtls](https://github.com/sublimelsp/LSP-jdtls).
 
+## JavaScript/TypeScript
+
+See also [Vue](#vue).
+
+There are multiple options:
+
+### Deno
+
+Follow installation instructions on [LSP-Deno](https://github.com/sublimelsp/LSP-Deno).
+
+### ESLint
+
+Follow installation instructions on [LSP-eslint](https://github.com/sublimelsp/LSP-eslint).
+
+### Flow
+
+Follow installation instructions on [LSP-flow](https://github.com/sublimelsp/LSP-flow).
+
+### TypeScript
```

Comment:
Maybe this header should also repeat the parent header (`JavaScript/TypeScript`)? It works for both.

Reply:
The tool used by LSP-typescript is called "TypeScript Language Server": https://github.com/typescript-language-server/typescript-language-server

If we rename the TypeScript subsection to JavaScript/TypeScript, should we also rename the Flow subsection to JavaScript/Flow?

Reply:
Well, it's hard to be fully precise and specific here so I guess it's not a big deal and we can leave as is.

`eslint` can also run on Vue files and possibly other with dedicated processors so it's not really possible to specify all the languages.